fortuna/src/index.js

```diff
@@ -1,46 +1,51 @@
-//@flow strict
```

Comment:
So do we need to manually flip the logic so the front-end can see their work using ```npm start```? This is just mostly a comment that should be answered than a complaint, since I get what you were thinking when you did this.

Reply:
Yeah, this was just temporary for testing and there just won't be a way to see my stuff in the real code. Once we have a navigation system, we won't need this.